### PR TITLE
find: add example to find and delete empty files

### PR DESCRIPTION
**Pull request by fazlearefin**
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).

### CODE REVIEW

**Comment by marchersimon** [comment]
Hey @fazlearefin, thanks for the great example. The only problem is, that pages can't have more than 8 examples. We could however remove another example. Maybe
```
- Find files matching a path pattern:

`find {{root_path}} -path '{{**/lib/**/*.ext}}'`
```
since it's very similar to the second one. But let's see what the other think.

**Comment by bl-ue** [comment]
I don't think we should remove that one, instead we could merge examples 2
```markdown
- Find files by matching multiple patterns:

`find {{root_path}} -name '{{*pattern_1*}}' -or -name '{{*pattern_2*}}'`
```
and 4
```markdown
- Find files matching a path pattern:

`find {{root_path}} -path '{{**/lib/**/*.ext}}'`
```
into this (replace example 2 with this)
```markdown
- Find files matching matching multiple path/name patterns:

`find {{root_path}} -path '{{**/path/**/*.ext}} -or -name '{{*pattern*}}'`
```
What do you guys think?

**Comment by CleanMachine1** [comment]
@fazlearefin Please remember to sign the agreement otherwise we can not merge your commits.

Also thank you for helping contributing with tldr, all help is wanted

File: pages/common/find.md
Comment (bl-ue) [inline pages/common/find.md:34]: I know you didn't write this @fazlearefin but this comma bothers me:
```suggestion
- Find files modified in the last 7 days and delete them:
```

**Comment by bl-ue** [comment]
@fazlearefin go ahead and make the changes — looks like @marchersimon & @CleanMachine1 agree with me on them.

**Review by fazlearefin — COMMENTED** [review]
Remove unnecessary comma

**Comment by bl-ue** [comment]
@fazlearefin if you're not sure about applying suggestions on GitHub, check out [this page](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes).

**Comment by CleanMachine1** [comment]
You need to take bl-ues merging of commands, since otherwise it will still be more than 8.
After you have done it I will give my approval and we can work on merging

**Comment by fazlearefin** [comment]
> You need to take bl-ues merging of commands, since otherwise it will still be more than 8.
> After you have done it I will give my approval and we can work on merging

Thanks. Done

**Review by bl-ue — APPROVED** [review]
Looks good!

**Comment by CleanMachine1** [comment]
Fine, approved!
Thanks for your first contribution

**Review by CleanMachine1 — APPROVED** [review]

File: pages/common/find.md
Comment (bl-ue) [inline pages/common/find.md:32]: I think maybe the hyphen should be out of the syntax...
```suggestion
`find {{root_path}} -mtime -{{7}} -delete`
```

Comment (CleanMachine1) [reply pages/common/find.md:32]: Yes, it should be changed so good spot bl-ue

File: pages/common/find.md
Comment (marchersimon) [inline pages/common/find.md:10]:
```suggestion
- Find files matching multiple path/name patterns:
```
That's too many `matching`s 😄

File: pages/common/find.md
Comment (marchersimon) [inline pages/common/find.md:36]:
```suggestion
`find {{root_path}} -type {{f}} -empty -delete`
```

File: pages/common/find.md
Comment (CleanMachine1) [reply pages/common/find.md:10]: it should be 
```suggestion
- Find files by matching multiple path/name patterns:
```

Comment (bl-ue) [reply pages/common/find.md:10]: I prefer the first suggestion.  @CleanMachine1 while technically correct, your suggestion makes it sound as though we were talking about _how_ `find` internally works for this invocation.

Comment (bl-ue) [reply pages/common/find.md:10]: This is a funny typo, and I'm surprised I didn't notice! 🙂

**Comment by bl-ue** [comment]
@fazlearefin don't worry about all of these suggestions — this is _the_ typical review process.  Since the beginning of the project, pages have accumulated from the community.  But in the past two or three years we've been getting a bit more strict on enforcing this and that style, but lots of the pages (espcially for the common tools e.g. `find`) still have issues, and these we notice when we're reviewing a new PR modify one of those pages.

**Comment by CleanMachine1** [comment]
Yes, it does seem intimidating. I didn't like the thought of all of the judgement, however after thinking about it a little, it makes perfect sense, its not a judgement of your work, its just keeping standards and stopping outliers

**Comment by fazlearefin** [comment]
Thanks. Yes, I understand completely. The goal is to produce the highest quality documentation possible.

**Comment by CleanMachine1** [comment]
Ok, this will probably be merged in the next 24 hours, we just wait for approval from others to give them a chance to spot any errors

**Comment by CleanMachine1** [comment]
Could you please commit the error fixes

File: pages/common/find.md
Comment (marchersimon) [reply pages/common/find.md:10]: I'm surprised we all didn't notice 😄

**Review by CleanMachine1 — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @fazlearefin

**Comment by bl-ue** [comment]
I don't think we need @sbrl's approval.  @CleanMachine1 go ahead and merge 👍🏻

**Comment by CleanMachine1** [comment]
Would you rebase or squash, I assume squash

**Comment by bl-ue** [comment]
Squash.

**Comment by CleanMachine1** [comment]
@fazlearefin Hope you plan on commiting some more

**Comment by bl-ue** [comment]
@CleanMachine1 95% of the time it's going to be squash.  Rebase only when the commits look nice and clean, and logical.  Usually that's only when there's a mass PR (e.g. a bunch of translations or similar modifications, one per page)...

**Comment by bl-ue** [comment]
>@fazlearefin Hope you plan on commiting some more

Yes @fazlearefin that would be great! However, don't feel pressured; every bit of your contributions are volunteered and you're bound by nothing to contribute, it's all optional.

**Pull request merged**
